usagov/usagov-2021
# Usagov 43 image webp

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-43

## Description

Added the navigation banner field to navigation pages. 
Created a new webp image style. 
Applied the webp image style to the banner images and the images in the navigation cards. 

## Type of Changes

  - Add screenshot showing what it should look like
  - Looks the same 

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Import config 
- Go to localhost/life-events 
-- Check that banner image is a smaller size than the one on prod (< 588kB); similarly check the thumbnails 
-- If no photos show refresh the page. This can happen on first load but won't happen on the static site (similar to the homepage) 
- Go to a random nav page (i.e localhost/taxes) 
- Download navigation banner and  than click edit
- Remove the first line from the body i.e `<section class="usa-hero usagov-hero" aria-label="Introduction" style="background-image: url(/themes/custom/usagov/images/topics/Banner_img_Benefits.png)"`>
- Upload and insert a new navigation banner image (the same image you downloaded) 
- Check that image displayed is smaller than image you downloaded 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
